gulpfile.js

```diff
@@ -109,6 +110,9 @@ const _genDist = ({minify = false} = {}) =>
             // convert JSON files to ES6 modules, so they can be included in Rollup bundle
             rollupJson(),
 
+            // gives rollup ability to read typescript files
+            rollupTypescript(),
```

Comment:
Interesting! How was this working before????

Reply:
I am not really sure about that. It was in out package.json so maybe it was implied.